### PR TITLE
Us08 saja alsheikh

### PR DESCRIPTION
**Pull request by sajawii**
1. create test US 08
2. combind doublecated POM files pages

### CODE REVIEW

**Review by hamidalakbarov — APPROVED** [review]
Looks great Saja, Thanks!

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java [hunk 1]
```diff
@@ -64,4 +68,50 @@ public void verify_the_user_see_the_following_modules( List<String> expectedModu
         Assert.assertEquals(expectedModules,actualModules);
 
     }
+
+    //Saja US
+    FilePage filePage = new FilePage();
+    Actions actions = new Actions(Driver.getDriver());
+    @Given("user on the dashboard page")
+    public void user_on_the_dashboard_page() {
+        Driver.getDriver().get("https://qa.meetsky.net");
```
Comment (DanSpicica) [inline src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java]: Saja for logging in with valid credentials we have a method in the LoginPage that you can use...

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java [hunk 2]
```diff
@@ -64,4 +68,50 @@ public void verify_the_user_see_the_following_modules( List<String> expectedModu
...
+        loginPage.userNameInputBox.sendKeys("User1");
+        loginPage.passwordInputBox.sendKeys("Userpass123");
+        loginPage.loginButton.click();
+
+    }
+    @When("the user clicks the {string} module")
+    public void the_user_clicks_the_module(String File) {
```
Comment (DanSpicica) [inline src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java]: For this also is a method in Base page and you can call it through the dashboardPage

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java [hunk 3]
```diff
@@ -64,4 +68,50 @@ public void verify_the_user_see_the_following_modules( List<String> expectedModu
...
+      filePage.File.click();
+    }
+    @When("user clicks the add icon on the top")
+    public void user_clicks_the_add_icon_on_the_top() {
+        actions.moveToElement(filePage.AddIcon);
+        filePage.AddIcon.click();
+        BrowserUtils.sleep(3);
```
Comment (DanSpicica) [inline src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java]: I see you've used a lot sleep method, was it just for you to see the flow and you forgot to take it off?

**Review by DanSpicica — APPROVED** [review]

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java (hunk 1 above)
Comment (sajawii) [reply src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java]: Hi Dan this is Hamid Step i used.

**Pull request closed without merging**